neon_audio/utils.py

```diff
@@ -62,9 +63,20 @@ def _plugin_to_package(plugin: str) -> str:
     }
     return known_plugins.get(plugin) or plugin
 
+def build_extra_dependency_list(config: Configuration, module: str, additional: List[str] = []) -> str:
```

Comment:
`config` here can just be a `dict`

`module` can also be assumed to be `audio` since this method is specific to the audio module with the `tts` config parsing

`additional` appears unused

Reply:
Additional is from cli.py:
```@click.option("--package", "-p", default=[], multiple=True,
              help="Additional package to install (can be repeated)")```

Reply:
not sure why I couldn't read earlier today, but I do see now that is valid